src/Octoshift/Services/OctoLogger.cs

```diff
@@ -30,6 +31,11 @@ public class OctoLogger
 
     private const string GENERIC_ERROR_MESSAGE = "An unexpected error happened. Please see the logs for details.";
 
+    private readonly HashSet<Regex> _redactionPatterns = new()
```

Comment:
two things here:
1. The `HashSet` will make no difference here. It's better to use lighter types like an `Array` or `List`.
2. Even though it's trivial, I think it's easier to just keep a list (or array) of strings rather than actual regex objects. It's less allocation and easier to maintain.

Reply:
Addressed in 342be5b